### PR TITLE
Final touches

### PR DESCRIPTION
**Pull request by TorHou**

### CODE REVIEW

**Comment by bgruening** [comment]
@TorHou ping

File: tools/peakachu/peakachu.xml [hunk 1]
```diff
@@ -101,11 +103,20 @@
...
-                    <option value="manual">Manual</option>
-                    <option value="none">None</option>
-                </param>
+                <conditional name="norm_method">
+                    <param name="norm_method_selector" type="select" label="Normalisation Method.">
+                        <option value="deseq" selected="True">DESeq2</option>
+                        <option value="manual">Manual</option>
+                        <option value="none">None</option>
+                    </param>
+                    <when value="manual">
```
Comment (bgruening) [inline tools/peakachu/peakachu.xml]: was passiert bei DESeq2 and None?

File: tools/peakachu/peakachu.xml [hunk 2]
```diff
@@ -114,22 +125,26 @@
...
-                </param>
+                <conditional name="norm_method">
+                    <param name="norm_method_selector" type="select" label="Normalisation Method.">
+                        <option value="tmm" selected="True">TMM</option>
+                        <option value="deseq">DESeq2</option>
+                        <option value="count">Count</option>
+                        <option value="manual">Manual</option>
+                        <option value="none">None</option>
+                    </param>
+                    <when value="manual">
```
Comment (bgruening) [inline tools/peakachu/peakachu.xml]: and here as well, what happens in the other cases?
A <when/> is at least needed

Comment (TorHou) [reply tools/peakachu/peakachu.xml]: Then everything is done internally by Peakachu, you don't need to provide your own normalization factors

Comment (TorHou) [reply tools/peakachu/peakachu.xml]: when it's `None` it won't normalize the counts ... so no additional info is needed

Comment (bgruening) [reply tools/peakachu/peakachu.xml]: Then please include the closing when.

Comment (TorHou) [reply tools/peakachu/peakachu.xml]: I'm not sure I understand ... see line 142 ?

**Pull request merged**